### ISSUE TITLE
0.3.0 freezes on start

### ISSUE THREAD

**Issue by azazar** [jwilm/alacritty#2333]
I'm using Ubuntu 18.04 Xfce. When I start Alacritty 0.3.0, it freezes after displaying window without drawing any contents or sending anything to stdout/stderr. 0.3.0 RC 3 works.

**Comment by azazar** [comment]
After installing 0.3.0 RC 3 and replacing it with 0.3.0 it stopped freezing.

**Comment by chrisduerr** [comment]
If you run Alacritty from a different terminal with `alacritt -vvv`, what output do you get and where does it get stuck?

**Comment by benjione** [comment]
Same issue, but only after first time opening after installation and update. `alacritty -vvv` does not hang and output is:

`[2019-04-20 10:31] [INFO] Welcome to Alacritty
[2019-04-20 10:31] [INFO] Configuration loaded from "/home/.../.config/alacritty/alacritty.yml"
[2019-04-20 10:31] [INFO] Device pixel ratio: 1
[2019-04-20 10:31] [INFO] Width: 1024, Height: 768
[2019-04-20 10:31] [INFO] Initializing glyph cache...
[2019-04-20 10:31] [INFO] ... finished initializing glyph cache in 0.062455869s
[2019-04-20 10:31] [INFO] Width: 724, Height: 412
[2019-04-20 10:31] [INFO] Cell Size: (9 x 17)
[2019-04-20 10:31] [INFO] Padding: (2 x 2)
[2019-04-20 10:31] [INFO] PTY Dimensions: Line(24) x Column(80)
[2019-04-20 10:31] [TRACE] [src/term/color.rs:30] Scaling RGB by 0.66 from Rgb { r: 238, g: 238, b: 238 } to Rgb { r: 157, g: 157, b: 157 }
[2019-04-20 10:31] [TRACE] [src/term/color.rs:97] Using config-provided dim colors
[2019-04-20 10:31] [INFO] Initialisation complete
[2019-04-20 10:31] [INFO] Font size changed to Size(22) with DPR of 1
[2019-04-20 10:31] [DEBUG] Resizing terminal
[2019-04-20 10:31] [DEBUG] New num_cols is 113 and num_lines is 44
[2019-04-20 10:31] [INFO] Width: 1024, Height: 768
[2019-04-20 10:31] [TRACE] [src/term/mod.rs:2014] Setting attribute: Bold
[2019-04-20 10:31] [TRACE] [src/term/mod.rs:2014] Setting attribute: Foreground(Named(Green))
[2019-04-20 10:31] [TRACE] [src/term/mod.rs:2014] Setting attribute: Reset
[2019-04-20 10:31] [TRACE] [src/term/mod.rs:2014] Setting attribute: Bold
[2019-04-20 10:31] [TRACE] [src/term/mod.rs:2014] Setting attribute: Foreground(Named(Blue))
[2019-04-20 10:31] [TRACE] [src/term/mod.rs:2014] Setting attribute: Reset
[2019-04-20 10:31] [TRACE] [src/term/mod.rs:1689] Carriage return
[2019-04-20 10:31] [TRACE] [src/term/mod.rs:1868] Clearing line: Right
[2019-04-20 10:31] [TRACE] [src/term/mod.rs:2014] Setting attribute: Bold
[2019-04-20 10:31] [TRACE] [src/term/mod.rs:2014] Setting attribute: Foreground(Named(Green))
[2019-04-20 10:31] [TRACE] [src/term/mod.rs:2014] Setting attribute: Reset
[2019-04-20 10:31] [TRACE] [src/term/mod.rs:2014] Setting attribute: Bold
[2019-04-20 10:31] [TRACE] [src/term/mod.rs:2014] Setting attribute: Foreground(Named(Blue))
[2019-04-20 10:31] [TRACE] [src/term/mod.rs:2014] Setting attribute: Reset
[2019-04-20 10:31] [DEBUG] Resizing terminal
[2019-04-20 10:31] [DEBUG] New num_cols is 80 and num_lines is 24
[2019-04-20 10:31] [INFO] Width: 724, Height: 412
[2019-04-20 10:31] [TRACE] [src/term/mod.rs:1689] Carriage return
[2019-04-20 10:31] [TRACE] [src/term/mod.rs:1868] Clearing line: Right
[2019-04-20 10:31] [TRACE] [src/term/mod.rs:2014] Setting attribute: Bold
[2019-04-20 10:31] [TRACE] [src/term/mod.rs:2014] Setting attribute: Foreground(Named(Green))
[2019-04-20 10:31] [TRACE] [src/term/mod.rs:2014] Setting attribute: Reset
[2019-04-20 10:31] [TRACE] [src/term/mod.rs:2014] Setting attribute: Bold
[2019-04-20 10:31] [TRACE] [src/term/mod.rs:2014] Setting attribute: Foreground(Named(Blue))
[2019-04-20 10:31] [TRACE] [src/term/mod.rs:2014] Setting attribute: Reset`

Hope that helps!

**Comment by chrisduerr** [comment]
This looks like the terminal is actually doing things, but they just aren't getting rendered.

When you click inside the window, or click and drag (to start a selection), can you get things to show up?

Some more hardware information would also be interesting, primarily GPU and driver.

**Comment by benjione** [comment]
@chrisduerr This was actually the output of alacritty -vvv when it was working, I can not reproduce the error when it hung in first place. It only hang at the first time I opened alacritty, but a reinstallation does not reproduce this bug. I will try to reproduce the bug. Maybe it is an issue at reading a configuration which is obsolete? 
Anyhow, it has to do with something that is done the first time alacritty is started.

**Comment by chrisduerr** [comment]
Sounds like this is a duplicate of https://github.com/jwilm/alacritty/issues/2303.
